install.ps1

```diff
@@ -0,0 +1,93 @@
+#Requires -Version 7.0
+
+param (
+    [Parameter(Mandatory = $false)] [System.Management.Automation.SemanticVersion]$DaggerVersion,
+    [Parameter(Mandatory = $false)] [string]$InstallPath = $env:HOMEPATH + '\dagger',
+
+    [Parameter(Mandatory = $false)] [System.Boolean]$InteractiveInstall = $false
+)
+
+# ---------------------------------------------------------------------------------
+# Author: Alessandro Festa
+# Co Author: Brittan DeYoung
+# Dagger Installation Utility for Windows users
+# ---------------------------------------------------------------------------------
```

Comment:
This script works fine for me. Now just need the `dagger.exe` that it downloads to give some output. That's the other PR :)